video/buffered_frame_decryptor.cc

```diff
@@ -63,15 +63,19 @@ BufferedFrameDecryptor::FrameDecision BufferedFrameDecryptor::DecryptFrame(
     return FrameDecision::kStash;
   }
   // When using encryption we expect the frame to have the generic descriptor.
-  if (frame->GetRtpVideoHeader().generic == absl::nullopt) {
-    RTC_LOG(LS_ERROR) << "No generic frame descriptor found dropping frame.";
-    return FrameDecision::kDrop;
-  }
+  // if (frame->GetRtpVideoHeader().generic == absl::nullopt) {
+  //   RTC_LOG(LS_ERROR) << "No generic frame descriptor found dropping frame.";
+  //   return FrameDecision::kDrop;
+  // }
```

Comment:
This prevents an Decryptor from being used if you don't have the frame header extension.  But you don't need the frame header extension, so remove this.